src/spyglass/common/common_dandi.py

```diff
@@ -0,0 +1,175 @@
+import os
+from uuid import uuid4
+
+import dandi.organize
+import dandi.validate
+import datajoint as dj
+import fsspec
+import h5py
+import pynwb
+from dandi.consts import known_instances
+from dandi.dandiapi import DandiAPIClient
+from dandi.validate_types import Severity
+from fsspec.implementations.cached import CachingFileSystem
+
+from spyglass.common.common_lab import LabMember
+from spyglass.common.common_usage import Export
+from spyglass.settings import export_dir
+from spyglass.utils.dj_helper_fn import _resolve_external_table
+from spyglass.utils.dj_mixin import SpyglassMixin
+
+dev_instance = known_instances["dandi-staging"]
```

Comment:
This isn't something that should be config dependent, since when accessing the data it needs to match where the upload was to (analagous to the `kachery_zone` in `AnalysisNwbfileKachery` table) . If we think that users will regularly want to be uploading to something other than the dandi production server I think it should be tracked as a key in `DandiPath`